x-pack/legacy/plugins/ml/public/jobs/jobs_list/components/jobs_list/jobs_list.js

```diff
@@ -148,7 +160,8 @@ class JobsListUI extends Component {
               )}
             data-row-id={item.id}
           />
-        )
+        ),
+        width: '3%'
```

Comment:
Could you explain in a comment what made the %-widths necessary compared to the previous version?

Reply:
I'll add in a comment explaining that the previous version depends on eui class overrides to set the width explicitly. These widths were added for use in the management table since. The jobs table in the ML plugin won't change as the overwritten class styles take precedence.
If/when we move the styles from those overwritten classes to always use props we'll likely have to change these values.

Reply:
Added comment in 8cd1f7d0d9dda2e069313683d9584dfac6095385